ex_2_ch_2.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jun 14 08:47:40 2018
+
+@author: amyhaddad
+"""
+
+string_1 = input("What is the input string? ")
+
+print(f"{string_1} has {len(string_1)} characters.")
```

Comment:
Perfect use of a `function` here! It does some work from you under the hood to calculate the length.